src/nifgen/examples/nifgen_script.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+import argparse
+import nifgen
+import numpy as np
```

Comment:
I'm not a fan of having our examples depend on large third party dependencies.
If we're using bumpy and scipy for trivial things, I'd rather not use them.